web/controllers/users_controller.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Notox.UserController do
+  use Notox.Web, :controller
+  alias Notox.User
+
+  def index(conn, _params) do
+    users = Repo.all(User)
+    render conn, :index, users: users
+  end
+
+  def show(conn, %{"id" => id}) do
+    user = Repo.get(User, id)
+    render conn, :show, user: user
+  end
+
+  def new(conn, _params) do
+    changeset = User.changeset(%User{})
+    render conn, :new, changeset: changeset
+  end
+
+  def create(conn, %{"user" => user_params}) do
+    changeset = User.registration_changeset(%User{}, user_params)
+    case Repo.insert(changeset) do
+      {:ok, user} ->
+        conn
+        |> Notox.Auth.login(user)
+        |> put_flash(:info, "User created!")
+        |> redirect(to: user_path(conn, :index))
+      {:error, changeset} ->
+        render(conn, :new, changeset: changeset)
+    end
+  end
+
+  def edit(conn, %{"id" => id}) do
```

Comment:
Controllers and models don't have to have a 1-to-1 mapping. In fact, many times in _real_ applications (more than just a couple [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) pages), they don't.

The users controller, however, does map directly to a model. Thus, I'd probably expect the `edit` action to be able to take an `id` and edit whatever user has that `id`. Even if you aren't doing that now, you might want to do that in the future, so you'd want to leave that action open for that possibility.

What you want to do _now_ is edit the current user. This is different than `users#edit` (which I said I would expect to edit an arbitrary user). It doesn't take an `id` and _always_ edits the logged in user. Since it's a specific behavior aside from just editing a user, I'd probable call it something different. As I said before, either `profile` or `account` (maybe account in this case, if its just the username and password? - I'd call it `profile` if you expected it to grow to allow editing a bio/avatar/etc).